Update end of year post and team assignments (#344)
2i2c-org/2i2c-brochure: content/authors/april-johnson/_index.md

---
# Display name
title: "April Johnson"

# Username (this should match the folder name and the name on publications)
authors:
- "april-johnson"

# Is this the primary user of the site?
superuser: false

# Role/position (e.g., Professor of Artificial Intelligence)
role: People Operations Lead

# Organizations/Affiliations
organizations:
- name: 2i2c
  url: "https://2i2c.org"
- name: April Johnson Coaching & Consulting
  url: "https://www.apriljohnson.io/"

# Short bio (displayed in user profile at end of posts)
bio:

# List each interest with a dash
interests: []

# Social/Academic Networking
# For available icons, see: https://sourcethemes.com/academic/docs/page-builder/#icons
#   For an email link, use "fas" icon pack, "envelope" icon, and a link in the
#   form "mailto:your-email@example.com" or "#contact" for contact widget.
social:
- icon: envelope
  icon_pack: fas
  link: 'mailto:april@2i2c.org'

# Enter email to display Gravatar (if Gravatar enabled in Config)
# email: ""

# Organizational groups that you belong to (for People widget)
#   Set this to `[]` or comment out if you are not using People widget.
user_groups:
- Delivery Enablement Team
---

> April brings ideas to life, whether that's making big change happen or creating teams from scratch. She has helped people make their businesses greener, design and build their technology and talent organizations, go public, incorporate new capabilities through acquisition, and scale up for the future. (from April's [LinkedIn profile](https://www.linkedin.com/in/apriljohnson/))

Reviewer: ```suggestion
April is a people & transformation consultant who brings ideas to life, whether that's making big change happen or creating teams from scratch. She has helped people make their businesses greener, design and build their technology and talent organizations, go public, incorporate new capabilities through acquisition, and scale up for the future. 
```
